document/core/util/macros.def

```diff
@@ -1280,6 +1290,7 @@
 .. |vdashglobalinst| mathdef:: \xref{appendix/properties}{valid-globalinst}{\vdash}
```

Comment:
line 1283 (above): `.. |vdashval| mathdef:: \xref{appendix/properties}{valid-val}{\vdash}`

should be `exec/modules` rather than `appendix/properties` (I think)

Reply:
Indeed, fixed.